lnrpc/routerrpc/router_backend.go

```diff
@@ -169,6 +169,11 @@ func (r *RouterBackend) QueryRoutes(ctx context.Context,
 		ignoredPairs[pair] = struct{}{}
 	}
 
+	cltvLimit, err := ValidateCLTVLimit(in.CltvLimit, r.MaxTotalTimelock)
```

Comment:
I am in doubt about this now. It is possible to call `QueryRoutes` with a different source pubkey. This allows the construction of routes not starting from ourselves. Maybe we should allow higher values, but if none is given, default to the max?

Reply:
Good point, fixed.